udata_event_service/producer.py

```diff
@@ -0,0 +1,40 @@
+import json
+import logging
+import os
+
+from kafka import KafkaProducer
+
+
+class KafkaProducerSingleton:
+    __instance = None
+
+    @staticmethod
+    def get_instance(kafka_uri: str) -> KafkaProducer:
+        if KafkaProducerSingleton.__instance is None:
+            kafka_api_version = os.environ.get("KAFKA_API_VERSION", "2.5.0")
+            KafkaProducerSingleton.__instance = KafkaProducer(
+                bootstrap_servers=kafka_uri,
+                value_serializer=lambda v: json.dumps(v).encode("utf-8"),
+                api_version=tuple(
+                    [int(value) for value in kafka_api_version.split(".")]
+                ),
+            )
+        return KafkaProducerSingleton.__instance
+
+
+def produce(
+    kafka_uri: str,
+    topic: str,
+    service: str,
+    key_id: str,
+    document: dict = None,
+    meta: dict = None,
+) -> None:
+    if kafka_uri:
+        producer = KafkaProducerSingleton.get_instance(kafka_uri)
```

Comment:
See this [issue](https://github.com/etalab/udata-event-core/issues/2) to improve producer logic based on kafka_uri.

We can deal this in another iteration.